solutions/gold/sumdiv.mdx

```diff
@@ -0,0 +1,72 @@
+## Explanation
```

Comment:
uh you forgot to add the yaml frontmatter

Reply:
Done